### PR TITLE
Unity 4.11.0, EmbeddedWallet -> InAppWallet + fix 4.10 regression

### PR DESCRIPTION
**Pull request by 0xFirekeeper**
## PR-Codex overview
This PR updates Unity to version 4.11.0, renames `EmbeddedWallet` to `InAppWallet`, fixes a regression, and refactors wallet handling.

### Detailed summary
- Updated Unity to version 4.11.0
- Renamed `EmbeddedWallet` to `InAppWallet`
- Refactored wallet handling by using `SUPPORTED_WALLET_IDS` array
- Replaced `PossibleWallet` type with `string` for wallet identifiers

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: packages/unity-js-bridge/src/thirdweb-bridge.ts [hunk 1]
```diff
@@ -66,30 +65,28 @@ const bigNumberReplacer = (_key: string, value: any) => {
...
 
-const WALLETS = [
-  MetaMaskWallet,
-  InjectedWallet,
-  WalletConnect,
-  CoinbaseWallet,
-  LocalWallet,
-  SmartWallet,
-  EmbeddedWallet,
+const SUPPORTED_WALLET_IDS = [
```
Comment (jnsdls) [inline packages/unity-js-bridge/src/thirdweb-bridge.ts:68]: great change actually!

File: packages/unity-js-bridge/src/thirdweb-bridge.ts [hunk 2]
```diff
@@ -335,13 +333,13 @@ class ThirdwebBridge implements TWBridge {
...
       // local wallet needs to be generated or loaded before connecting
-      if (walletInstance.walletId === walletIds.localWallet) {
+      if (wallet === "localWallet") {
         await this.initializeLocalWallet(password as string);
         walletInstance.connect({ chainId: chainIdNumber });
       }
 
-      if (walletInstance.walletId === walletIds.embeddedWallet) {
-        const embeddedWallet = walletInstance as EmbeddedWallet;
+      if (wallet === "inAppWallet") {
```
Comment (jnsdls) [inline packages/unity-js-bridge/src/thirdweb-bridge.ts:341]: what if something still sends `embedded` ? is that possible or not possible due to how unity is set up?

**Review by jnsdls — APPROVED** [review]
LGTM, 1 small question but overall lets go

File: packages/unity-js-bridge/src/thirdweb-bridge.ts (hunk 2 above)
Comment (0xFirekeeper) [reply packages/unity-js-bridge/src/thirdweb-bridge.ts:341]: not gonna be possible after 4.11, just trying to get rid of dependency on previous walletIds thing

**Pull request merged**